### ISSUE TITLE
Black Screen on Running Program

### ISSUE THREAD

**Issue by Recon419A** [yhatt/marp#74]
When I run the program (both from 0.0.8 and building from source), I get a black screen. Unfortunately, I'm having issues with screen shots right now as well, so I can't paste it, but the terminal does print like a thousand of these:

`[electron] [7373:0812/112208:ERROR:gles2_cmd_decoder.cc(2167)] [.CommandBufferContext.RenderWorker-0x16244b262140]GL ERROR :GL_INVALID_VALUE : ScopedTextureBinder::dtor: <- error from previous GL command
`

**Comment by yhatt** [comment]
It seems to be caused by compatibility between GPU and Chromium. (e.g. [Similar case of Ubuntu Chromium](http://askubuntu.com/questions/793301/chromium-launches-to-a-black-screen-also-sudo-apt-get-update-returns-errors))

If you are ready to build Marp from source, could you try to add `app.disableHardwareAcceleration()` to [`coffee/main.coffee:49`](https://github.com/yhatt/marp/blob/master/coffee/main.coffee#L49)? As below:

```coffee
app.on 'ready', ->
  app.disableHardwareAcceleration()

  global.marp.mainMenu = new MainMenu
    ...
```

We would have worth to support to disable GPU acceleration if this issue would improved in above.

**Comment by Recon419A** [comment]
Sorry to take so long getting back.

When I made that change, I got different behavior. `npm start` produced a popup with the following error:

`
A JavaScript error occurred in the main process

Uncaught Exception:
Error: app.disableHardwareAcceleration() can only be called before app is ready
    at Error (native)
    at EventEmitter.<anonymous> (/home/drew/apps/marp/js/main.js:1:1161)
    at emitOne (events.js:101:20)
    at EventEmitter.emit (events.js:188:7)
`

It looks like it doesn't want me to call it there.
